react-native-scripts/template/README.md

```diff
@@ -142,6 +143,47 @@ You can optionally use a [plugin for your IDE or editor](https://flow.org/en/doc
 
 To learn more about Flow, check out [its documentation](https://flow.org/).
 
+## Adding ESLint
+
+[ESLint](http://eslint.org/) is a configurable javascript linter. ESLint can help standardize a style guide across the team.
+
+To add eslint to a Create React Native App project, follow these steps:
+
+1. Run `npm install --save-dev eslint eslint-plugin-react`
```

Comment:
people should probably use react-native preset, or eslint-config-react-native-prettier